tutorials/clinical/20_seeg.py

```diff
@@ -42,6 +43,8 @@
 
 print(__doc__)
 
+np.set_printoptions(suppress=True)  # suppress scientific notation
```

Comment:
is this really necessary?

Reply:
I'm okay with having this, but we need to update `reset_warnings` to `np.set_printoptions` to whatever they are when `conf.py` is initially loaded (which can be obtained before any examples are run directly in `conf.py` using `get_printoptions` presumably)

Reply:
I have a hard time understanding the affines in scientific notation. I'm not sure what you mean @larsoner, is that just like putting it back at the end?

Also that's how `nibabel` does it and it looks nice.